cmd/relay-gossip/main.go

```diff
@@ -144,23 +157,40 @@ var clientCmd = &cli.Command{
 			return xerrors.Errorf("constructing host: %w", err)
 		}
 
-		c, err := client.NewWithPubsub(ps, nil, nil)
+		chainHash, err := hex.DecodeString(cctx.String("chain-hash"))
```

Comment:
can this not be optional?

Reply:
Soon! It's one of my tasks for this week.